src/editing.md

```diff
@@ -58,7 +58,7 @@ which you can run periodically.
 ### Effective Learning
 
 Different people like to review in different ways, but there are some
-general concepts to keep in mind. An excellent introduction is [this article](http://www.supermemo.com/articles/20rules.htm) on the SuperMemo
+general concepts to keep in mind. An excellent introduction is [this article](https://supermemo.guru/wiki/20_rules_of_knowledge_formulation) on the SuperMemo
```

Comment:
I'm afraid I think the old link (https://www.supermemo.com/en/blog/twenty-rules-of-formulating-knowledge) actually is a better choice here - it goes into more detail about the listed points, and spends less time trying to advocate for incremental reading.

Reply:
Thank you for your reply.

Understood.

Within the link I have supplied (https://supermemo.guru/wiki/20_rules_of_knowledge_formulation), there is another link (let's call it **Link3**): http://super-memory.com/articles/20rules.htm

**Link3** has a "See also" section at the very bottom, which includes:
1. https://super-memory.com/english/ol/ks.htm: this has FAQ and other useful examples
2. https://super-memory.com/help/faq/ks.htm: this has advanced concepts such as the [importance of redundancy](https://super-memory.com/english/ol/ks.htm#Redundancy) and the distinction between [content semantics and wording](https://super-memory.com/english/ol/ks.htm#Wording).

So, how about including Link3 instead of https://www.supermemo.com/en/blog/twenty-rules-of-formulating-knowledge?



HR

Reply:
Sure, that works for me.

Reply:
Fixed!